server/routes/api/article.js

```diff
@@ -16,9 +17,9 @@ const router = express.Router();
 router.post('/articles', checkToken, upload.single('imgUrl'), ArticleValidations.validateCreateArticle, ArticleController.createArticle);
 router.get('/articles/feed', ArticleController.getAllArticles);
 router.get('/articles', checkToken, ArticleController.getUserArticles);
-router.get('/categories', checkToken, CategoryController.getAllCategories);
-router.post('/categories', checkToken, CategoryController.createCategory);
```

Comment:
Remove authentication from this route. They are not needed. A user who is not login should be able to read articles by categories.